components/com_banners/src/Model/BannersModel.php

```diff
@@ -186,14 +186,16 @@ protected function getListQuery()
             } else {
                 $temp   = [];
                 $config = ComponentHelper::getParams('com_banners');
-                $prefix = $config->get('metakey_prefix');
+                $prefix = $config->get('metakey_prefix', '');
 
                 if ($categoryId) {
                     $query->join('LEFT', $db->quoteName('#__categories', 'cat'), $db->quoteName('a.catid') . ' = ' . $db->quoteName('cat.id'));
                 }
 
+                $isDbMySqlVersionFrom804 = $db->getServerType() === 'mysql' && version_compare($db->getVersion(), '8.0.4', '>=');
+
                 foreach ($keywords as $key => $keyword) {
-                    $regexp       = '[[:<:]]' . $keyword . '[[:>:]]';
+                    $regexp       = $isDbMySqlVersionFrom804 ? '\\b' . $keyword . '\\b' : '[[:<:]]' . $keyword . '[[:>:]]';
```

Comment:
```suggestion
                    $regexp       = $db->getServerType() === 'mysql' ? '\\b' . $keyword . '\\b' : '[[:<:]]' . $keyword . '[[:>:]]';
```
See my previous review comment.